### DISCUSSION TITLE
Host metrics for Appsignal

### DISCUSSION THREAD

**Discussion by zwippie** [dokku/dokku#4720] [Q&A] [answered]
I want to display _host metrics_ (load/mem usage) in Appsignal but no data is coming in.

According to the [appsignal docs](https://docs.appsignal.com/metrics/host-metrics/#dokku-support) the setup should be pretty much like a heroku setup, using a ['working Heroku Logplex Logdrain'](https://docs.appsignal.com/heroku/host-metrics.html).

This is where I'm stuck. _Is there a dokku equivalent of a 'logplex drain'?_ I've searched the plugins but could not find anything.

**Comment by josegonzalez** [comment]
I would tell the Appsignal folks that their docs are bad. Did they even try to set this up with Dokku? The `DYNO` variable is force-set on all containers, and cannot be removed.

The logplex drain they are talking about is something Heroku specific. We don't have an equivalent.

Maybe you want container metrics? https://docs.appsignal.com/metrics/host-metrics/containers.html

**Reply by zwippie** [reply]
Agreed their docs are minimal and possibly incorrect. I will ask them for more details and report back later.

**Comment by zwippie** [comment, marked as answer]
Ok, this seems to be fixed now in the latest releases of the AppSignal libraries, like the [appsignal-elixir](https://github.com/appsignal/appsignal-elixir/blob/main/CHANGELOG.md#220) package (v 2.2.0).

It will now report host metrics if the `DOKKU_ROOT` env var is set. Unfortunately this env var is only set when running the dokku script and may not exists in running containers, so I've added it manually with `dokku config:set DOKKU_ROOT=/home/dokku` and it works!
